tests/platforms/linux/system/test__find_changelog_filename.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from briefcase.platforms.linux.system import find_changelog_filename
+
+from ....utils import create_file
+
+
+def test_no_changelog(tmp_path):

```

Comment:
Tests should all have a docstring that explain what is being tested, however, obvious the title might be.